app/components/search-header.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
```

Comment:
Is there a reason to separate these into components? I don't think search page itself is going to be reusable.